vllm/model_executor/model_loader/weight_utils.py

```diff
@@ -135,18 +135,19 @@ def get_quant_config(model_config: ModelConfig,
     else:
         hf_folder = model_name_or_path
 
-    possible_config_filenames = quant_cls.get_config_filenames()
-
-    # If the quantization config is not found, use the default config.
-    if not possible_config_filenames:
-        return quant_cls()
-
     config_files = glob.glob(os.path.join(hf_folder, "*.json"))
+    possible_config_filenames = quant_cls.get_config_filenames()
 
     quant_config_files = [
         f for f in config_files if any(
             f.endswith(x) for x in possible_config_filenames)
     ]
+
```

Comment:
for GPTQ/AWQ, having a `quantize_config.json` is not necessarily required. So I think this check could break models that have:

- `quantization_config` in `config.json`
- no `quantize_config.json`

For example: https://huggingface.co/casperhansen/llama-3-70b-instruct-awq/tree/main

Our CI does not seem to have any models with this setup

Reply:
Thanks, I fixed this now by removing the quantize_config.json support -- we can just use `config.json` for specifying the quantization for FP8 checkpoints for the time being :)